bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitImpl.java

```diff
@@ -199,9 +190,12 @@ private void startBridge() throws IOException {
 
     private void startHomekitServer() throws IOException {
         if (homekitServer == null) {
-            networkInterface = InetAddress.getByName(settings.networkInterface);
-            homekitServer = new HomekitServer(networkInterface, settings.port);
-            startBridge();
+            if (!mdnsClient.getClientInstances().isEmpty()) {
+                homekitServer = new HomekitServer(mdnsClient.getClientInstances().iterator().next(), settings.port);
```

Comment:
How do you know you are picking the right client instance?

Reply:
@cpmeister  as always, a good question. 
i was thinking that any instance would good enough as mDNS gets replicated across all instances. but after your comment, i had second thought and it is not much about mDNS instance but more about IP address of the mDNS instance which homekit also use for binding, i.e. where it expects requests from homekit client. this ip should accessible from iphone, etc

i see 2 options:
- bring back to homekit setting the IP address, which i use to filter mDNS. however, this IP is not always set correctly by user.
- use all mDNS clients, this would mean,  homekit will be available at all interfaces configured in openHAB

im tending to the last option. multiple interface is probably not that common case and it should not increase the memory footprint.